routing-framework/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM ubuntu:20.04
```

Comment:
Should we document this - (how to create the image and/or this entire new process?)

Reply:
Of course, but we didn't define this process yet :D

Reply:
remove